.circleci/config.yml

```diff
@@ -35,6 +35,15 @@ commands:
   ###
   # App commands
   ###
+  app-save-env-file:
+    steps:
+      - run:
+          name: "App: save env file"
+          command: |
+            cd app 
+            echo $APP_ENV | base64 -d > .env
```

Comment:
Is `APP_ENV` a CircleCI environment variable?

Reply:
Yes I set this one up similar to how @mhuynh5757 setup Google Services JSON in line 62

Reply:
Gotcha. I think this is a different situation where Google Services JSON is generated from google. But the .env file is generated by us. Instead of one single APP_ENV in circle ci, we should add each of the environment variables in circle ci instead. With that, you probably won't need to modify this config.yml